From 183dfa500a0e578fb05655f91f699fa09047a48d Mon Sep 17 00:00:00 2001
From: from minimyth2 <unknown@unknown.org>
Date: Thu, 18 Jan 2024 08:24:44 +0100
Subject: [PATCH 172/184] 
 1107-hack-hack-soc-rockchip-power-domain-workaround-for-oops-on-rk3528.patch

This is dirty hack needed to stop kernel oops on rk3528.
Root casue is under investigation....
---
 drivers/soc/rockchip/pm_domains.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/rockchip/pm_domains.c b/drivers/soc/rockchip/pm_domains.c
index 84bc022f9..46c317a50 100644
--- a/drivers/soc/rockchip/pm_domains.c
+++ b/drivers/soc/rockchip/pm_domains.c
@@ -456,7 +456,7 @@ static int rockchip_pd_power(struct rockchip_pm_domain *pd, bool power_on)
 			rockchip_pmu_save_qos(pd);
 
 			/* if powering down, idle request to NIU first */
-			rockchip_pmu_set_idle_request(pd, true);
+//			rockchip_pmu_set_idle_request(pd, true);
 		}
 
 		rockchip_do_pmu_set_power_domain(pd, power_on);
-- 
2.39.2

